docfx_yaml/extension.py

```diff
@@ -732,6 +732,52 @@ def extract_product_name(name):
     return product_name
 
 
+def _extract_type_name(annotation: Any) -> str:
```

Comment:
`Dict[Any]`?

Reply:
Ah, tried looking into the type and it's very messy lol it returns the inspected object's class directly. I will add this to the docstring!

FYI this is what I got from a small sample:
```
type: <class 'typing._CallableGenericAlias'>
type: <class 'typing._GenericAlias'>
type: <class 'typing._CallableGenericAlias'>
type: <class 'typing._GenericAlias'>
type: <class 'typing._CallableGenericAlias'>
type: <class 'typing._GenericAlias'>
type: <class 'typing._CallableGenericAlias'>
type: <class 'typing._GenericAlias'>
type: <class 'typing._CallableGenericAlias'>
type: <class 'typing._UnionGenericAlias'>
type: <class 'typing._CallableType'>
type: <class 'str'>
```